src/macros.rs

```diff
@@ -105,6 +105,22 @@ macro_rules! register_counter {
 }
 
 #[macro_export]
+macro_rules! register_counter_vec {
+    ( $ OPTS : expr , $ LABELS_NAMES : expr ) => {
+        {
+            let counter_vec = $crate::CounterVec::new($OPTS, $LABELS_NAMES).unwrap();
+            $crate::register(Box::new(counter_vec.clone())).map(|_| counter_vec)
+        }
+    };
+
+    ( $ NAME : expr , $ HELP : expr , $ LABELS : expr , $ LABELS_NAMES : expr ) => {
```

Comment:
why confusing? you can remove labels here, or rename LABELS to CONST_LABELS and LABELS_NAMES to VARIABLE_NAMES